src/components/PeerUpdate.tsx

```diff
@@ -180,11 +195,13 @@ const PeerUpdate = (props: any) => {
   useEffect(() => {}, [users]);
 
   const routeAddAllowed = (os: string): boolean => {
-    return os !== ""
-        && !os.toLowerCase().startsWith("darwin")
-        && !os.toLowerCase().startsWith("windows")
-        && !os.toLowerCase().startsWith("android")
-  }
+    return (
```

Comment:
did not touch that part of code.

Reply:
I understand. It is just got my attention because shown in diff. Would you mind changing to allow a list approach? Something like this
```
const allowList: string[] = ['linux'];
return allowList.includes(os.toLowerCase())
```

Reply:
done